README.md

```diff
@@ -1,21 +1,17 @@
 # D://
 _Bitcoin dynamic name protocol_
 
-
-
-
 ## Benefit
 A D:// transaction refers to another transaction/file and thus forms a layer which state can be overwritten. This way, referencing content via D:// links (instead of b:// or c://) has the advantage of keeping the URL while the content can change.
 
-Example B:// (**can not be changed** once set):<br>
-``<img src="B://<TxID(1)>"``
+Example B:// (**can not** be changed once set):<br>``<img src="B://<TxID>"``
 
-Example D:// (Pointer **can be changed**, because the state of the entry can be overwritten.):<br>  
-``<img src="D://<OwnerBitcoinAddress>/<key>"``
+Example D:// (Pointer **can** be changed, because the state can be updated):<br>``<img src="D://<OwnerBitcoinAddress>/<key>"``
 
+Please note the key should always be presented as URL encoded to the user.
 
 #### Overwrite D:// State
-New transactions with the same `key` from a sender automatically override the old state. Protocol API always outputs only the most current state
+New transactions with the same `key` from a sender overwrite the previous state. The PlProtocol API always outputs only the most current state
```

Comment:
PlProtocol -> typo

Reply:
Oh gosh - you are quick! I was hoping no one saw this so force pushed a fix :-)